R/0_imports.R

```diff
@@ -21,3 +21,6 @@ rsample::assessment
 
 #' @import vctrs
 NULL
+
+#' @import sf
```

Comment:
This is necessary for `boston_canopy` to print successfully. Without it, `pillar` causes an error because it doesn't understand the list column; as soon as `sf` is loaded it prints fine. I don't love importing the entire namespace but am not sure if there's a smarter way to go about this.